pkg/virt-config/config-map.go

```diff
@@ -34,17 +33,23 @@ import (
 	"kubevirt.io/kubevirt/pkg/util"
 )
 
-const featureGateEnvVar = "FEATURE_GATES"
-
 const (
-	dataVolumesGate   = "DataVolumes"
-	cpuManager        = "CPUManager"
-	liveMigrationGate = "LiveMigration"
-	SRIOVGate         = "SRIOV"
-	configMapName     = "kubevirt-config"
+	configMapName         = "kubevirt-config"
+	featureGateEnvVar     = "FEATURE_GATES"
+	emulatedMachineEnvVar = "VIRT_EMULATED_MACHINES"
 )
 
-func ParseFeatureGatesFromConfigMap() {
+func Load() {
```

Comment:
can you name this `init()`? It will be called automatically then exactly once, as soon as any other func in this package is called. So no need to manually call Load() anymore